docs/source/conf.py

```diff
@@ -175,3 +180,22 @@ def setup(app):
     app.add_config_value('m2r_disable_inline_math', False, 'env')
     app.add_directive('mdinclude', MdInclude)
     app.add_stylesheet('custom.css')
+
+    with open(
+        os.path.join(os.path.dirname(os.path.realpath(__file__)), 'component_docs.rst'),
+        'w',
+    ) as f:
+        f.write("labscript suite components\n")
+        f.write("==========================\n")
+        f.write("    \n")
+        f.write(".. toctree::\n")
+        f.write("    :maxdepth: 2\n")
+        f.write("    \n")
+        if project != "the labscript suite":
+            f.write(
+                "    labscript suite (metapackage)<{}>\n".format(
+                    intersphinx_mapping['labscript-suite'][0]
+                )
+            )
+        for ls_prog in labscript_suite_programs:
```

Comment:
Alphabetically sort contents of `labscript_suite_programs`?

Reply:
Do we want alphabetical or data flow order? 
Should the order of the side bar links match the order in the table on the same page?

Reply:
I swapped it to alphabetical a while ago so I guess I'll mark this as resolved.